### PR TITLE
feat: add new bridge network

### PR DESCRIPTION
**Pull request by antrusd**
This PR will add net `bridge` driver support. The driver almost similar with `lxc-user-nic` but without attaching connectivity between host namespace with target namespace. The connectivity need to be configured manually with the help `veth` interface (need root privilege) from host namespace which can be "watched" using `bash` script +  `systemd` service.

eg.
```bash
#!/usr/bin/env bash

export ROOTKIT_STATE_DIRECTORY=/path/to/rootkit/state/directory
export BRIDGE=docker0
export VETHP=docker0p1
export VETHC=docker0c1

# add masquerade support
echo 1 > /proc/sys/net/ipv4/ip_forward
iptables-save | grep -q ${BRIDGE} || iptables -t nat -A POSTROUTING -s 172.17.0.0/16 ! -o ${BRIDGE} -j MASQUERADE

while :; do
  until test -r ${ROOTKIT_STATE_DIRECTORY}/child_pid; do
    sleep 1
  done

  child_pid=$(cat ${ROOTKIT_STATE_DIRECTORY}/child_pid)
  mkdir -p /var/run/netns
  ln -sf /proc/${child_pid}/ns/net /var/run/netns/${BRIDGE}
  ip link add ${VETHC} type veth peer name ${VETHP}
  ip link set ${VETHP} up
  ip link set ${VETHP} master ${BRIDGE}
  ip link set ${VETHC} netns ${BRIDGE}

  while test -r ${ROOTKIT_STATE_DIRECTORY}/child_pid && ip link show ${VETHP} &>/dev/null; do
    sleep 1
  done
done
```

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
I'd rather prefer to have `rootlesskit --net=none` that would just leaves netns empty, as the connectivity has to be configured by an external script

File: Makefile [hunk 1]
```diff
@@ -9,6 +9,9 @@ all: $(addprefix bin/, $(BINARIES))
 clean:
 	$(RM) -r bin/ _artifact/
 
+bin/rootlesskit-debug: $(GO_FILES)
```
Comment (AkihiroSuda) [inline Makefile:12]: This should be a separate PR, and I'd rather prefer to use a variable like `GO_BUILD_FLAGS`

**Comment by antrusd** [comment]
Actually `none`  was the option I looking for in the first time, but forgot why I end up with `bridge` proposal. Will close this and will propose `none` network here https://github.com/rootless-containers/rootlesskit/pull/430 .

File: Makefile (hunk 1 above)
Comment (antrusd) [reply Makefile:12]: agree
